src/3d/qgsshadowrenderingframegraph.cpp

```diff
@@ -133,11 +133,54 @@ Qt3DRender::QFrameGraphNode *QgsShadowRenderingFrameGraph::constructPostprocessi
   mPostprocessClearBuffers = new Qt3DRender::QClearBuffers( mPostprocessPassLayerFilter );
   mPostprocessClearBuffers->setClearColor( QColor::fromRgbF( 0.0f, 0.0f, 0.0f ) );
 
+  //
+  {
```

Comment:
do we need the code to be inside the block?

Reply:
Not really needed, it is just to make sure that I only pick up mRenderCaptureTargetSelector from that context (previously I was using the same prefix in the objects names).